simplified-books-controller/src/main/java/org/nypl/simplified/books/controller/BookBorrowWithDefaultAcquisitionTask.kt

```diff
@@ -31,31 +36,60 @@ class BookBorrowWithDefaultAcquisitionTask(
   private val services: ServiceDirectoryType
 ) : Callable<TaskResult<BookStatusDownloadErrorDetails, Unit>> {
 
+  private val logger =
+    LoggerFactory.getLogger(BookBorrowWithDefaultAcquisitionTask::class.java)
   private val borrowStrings =
     this.services.requireService(BookBorrowStringResourcesType::class.java)
+  private val bookRegistry =
+    this.services.requireService(BookRegistryType::class.java)
+
+  /**
+   * The initial book value.
+   */
+
+  private val bookInitial: Book =
```

Comment:
You can drop the explicit type here since it can be inferred directly from the assignment.